replication/row_event.go

```diff
@@ -1182,6 +1217,29 @@ func decodeString(data []byte, length int) (v string, n int) {
 	return
 }
 
+func decodeStringLatin1(data []byte, length int) (v string, n int) {
```

Comment:
Might be good to make things a bit more generic, so it could work with multiple charsets and not just `latin1`.

However I think a vast majority of MySQL installations use either `latin1` or `utf8`/`utf8mb4`. Maybe some `ascii`, but that's a subset of both `latin1` and `utf8`/`utf8mb4`...